README.rst

```diff
@@ -186,3 +186,5 @@ Common problems and fixes
 =========================
 * libXp.so.6 missing (in ubuntu >= 15.10, for example)
 This is a known is http://askubuntu.com/questions/719839/libxp-so-6-missing-15-10. The idea is to install it manually from official sources https://launchpad.net/ubuntu/wily/+package/libxp6
+
+* Beware: the pathname for "scratch" (directory for cache) must not contain any space character! This triggered a nasty bug in nipype....
```

Comment:
I need to learn how to do better PR ;-)
I only have one branch on my fork of pypreprocess. Should I have created
different branches for different dev?

Chris


--
Christophe Pallier

Contact info: http://www.pallier.org/w/pmwiki.php/Main/Contact

On Sun, Jun 19, 2016 at 1:15 PM, DOHMATOB Elvis <notifications@github.com>
wrote:

> In README.rst
> <https://github.com/neurospin/pypreprocess/pull/231#discussion_r67615081>:
>
> > @@ -186,3 +186,5 @@ Common problems and fixes
> >  =========================
> >  * libXp.so.6 missing (in ubuntu >= 15.10, for example)
> >  This is a known is http://askubuntu.com/questions/719839/libxp-so-6-missing-15-10. The idea is to install it manually from official sources https://launchpad.net/ubuntu/wily/+package/libxp6
> > +
> > +* Beware: the pathname for "scratch" (directory for cache) must not contain any space character! This triggered a nasty bug in nipype....
>
> Ideally, this would go in a separate PR as it has nothing to do with spm12
> standalon installation or STC.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/neurospin/pypreprocess/pull/231/files/8caa896e8ea92a2478829bd93dbdbe53d287f26a#r67615081>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABdbMoFkN-AlBR_VpaTi2BpGIwM8hV9Tks5qNSTjgaJpZM4IqgU1>
> .
>